### PR TITLE
Error state Example

### PR DESCRIPTION
**Pull request by roideuniverse**

### CODE REVIEW

File: app/src/main/java/com/carrotcreative/recyclercore_example/MainActivity.java [hunk 1]
```diff
@@ -52,6 +75,7 @@ public void success(GithubUser[] githubUsers, Response response)
                     public void failure(RetrofitError error)
                     {
                         // TODO, handle
```
Comment (BrandonRomano) [inline app/src/main/java/com/carrotcreative/recyclercore_example/MainActivity.java]: Let's remove this TODO!

**Comment by BrandonRomano** [comment]
My only suggestion would be to possibly add an action button to force an error.

I had to go airplane mode to test out the error state, which is really fine but would be nicer if there was an in app way to force it

**Comment by roideuniverse** [comment]
Not a problem. Added that as an OptionsMenu.

**Comment by BrandonRomano** [comment]
:+1:  -- might want to wait to merge so you can repoint the submodule to recycler-core master after that's merged

**Comment by roideuniverse** [comment]
Updated. Good for merge!

**Pull request merged**